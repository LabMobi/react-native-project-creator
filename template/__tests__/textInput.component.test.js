import React from 'react';
import renderer from 'react-test-renderer';
import TextInput from '../src/components/common/textInput';

describe('textInput component', () => {
  it('renders textInput correctly', () => {
    const tree = renderer
      .create(<TextInput handleChangeDebounce={() => {}} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
