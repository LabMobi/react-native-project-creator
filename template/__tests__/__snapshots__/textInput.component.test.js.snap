// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`textInput component renders textInput correctly 1`] = `
<TextInput
  clearButtonMode="while-editing"
  onChangeText={[Function]}
  placeholderTextColor="rgb(28, 28, 30)"
  returnKeyType="done"
  selectionColor="rgb(0, 122, 255)"
  style={
    Array [
      Object {
        "borderRadius": 7,
        "borderWidth": 1,
        "height": 50,
        "paddingHorizontal": 10,
        "width": "95%",
      },
      Object {
        "backgroundColor": undefined,
        "borderColor": undefined,
        "color": "rgb(0, 122, 255)",
      },
      undefined,
    ]
  }
/>
`;
